README.md

```diff
@@ -53,6 +53,10 @@ Add the port number in the newly created user rc file. You should at least edit
 
     export SERVER_PORT=9000
 
+Add the API key to retrieve departure information from https://opentransportdata.swiss into the user rc file (keepass). For instance:
```

Comment:
or directly in `.bashrc`

Reply:
@loic nice idea with the guard function!